src/Control/Effect/Exception.hs

```diff
@@ -0,0 +1,254 @@
+{-# LANGUAGE DeriveFunctor, ExistentialQuantification, RankNTypes, StandaloneDeriving #-}
+-- | Operations from "Control.Exception" lifted into effectful contexts using 'Control.Effect.Lift.Lift'.
+--
+-- @since 1.0.0.0
+module Control.Effect.Exception
```

Comment:
Big shout out to @robrix here for doing the painful work of unlifting all these functions here ❤️